survey_creation/de_17/invitation.txt

```diff
@@ -22,24 +22,24 @@ Martin Hammitzsch und Stephan Janosch
 
 -------------------------------------------------------------
 
-Survey about research software in Germany and people involved
+Survey about research software in Germany, and the people involved in it
 
 Hello,
 
-Do you develop research software or you are involved in the scientific software creation process? Than we would be glad if you could participate in this year's survey: https://softwaresaved.limequery.com/000000?lang=en
+Do you develop research software, or are you involved in the scientific software creation process? Then we would very much like you to participate in the following survey: https://softwaresaved.limequery.com/000000?lang=en.
 
-Right now we don't have much knowledge about the community of software developing people in research and science. This survey should help to gather insights in order to support research funders and other institutions to develop strategies and funding programs as well as policies.
+As of now there is not much knowledge about the community of those in research and science who develop software. This survey aims to gain valuable insights into this community in order to support research funders and other institutions to develop strategies and funding programs as well as policies.
 
-This survey gives you the opportunity to include your view and thus to be part of the development.
+This survey gives you the opportunity to make your point of view and experiences be heard, and thus be part of the development of this community.
 
-In case you know others who develop software in research then feel free to forward this invitation. If you forward this mail to a mailing list, then please be so kind and put Martin or Stephan into CC.
+In case you know others who develop software in research, please feel free to forward this invitation. If you forward this mail to a mailing list, please be so kind and copy in Martin (*email address here!*) and Stephan (*email address here!*).
 
-The survey results will be published under CC BY-NC licence and published at the de-RSE mailing list [3] as well as on the de-RSE blog [4]. Right now similar surveys are done as well in UK, Canada, Australia, Norway, The Netherlands, USA and South Africa. For better comparison this survey was closely coordinated with the others.
+The survey results will be published under a CC-BY-NC license (*which one?*), and announced and evaluated via the de-RSE mailing list [3] and the de-RSE blog [4]. Simultaneously, similar surveys are conducted in the UK, Canada, Australia, Norway, the Netherlands, the USA and South Africa. For reasons of comparability, this survey was closely coordinated with the others.
```

Comment:
Streamlined with German version, but not sure if this is what you meant. See above.